elide-core/src/test/java/com/yahoo/elide/jsonapi/JsonApiTest.java

```diff
@@ -208,8 +208,8 @@ public void readSingle() throws IOException {
         assertEquals(data.getType(), "parent");
         assertEquals(data.getId(), "123");
         assertEquals(attributes.get("firstName"), "bob");
-        assertEquals(relations.get("children").getData().get().iterator().next().getType(), "child");
-        assertEquals(relations.get("children").getData().get().iterator().next().getId(), "2");
+        assertEquals(relations.get("children").getData().getSingleValue().getType(), "child");
```

Comment:
This works but doesn't make sense. Not sure why our doc indicates that `children` is a `toOne` field